book/scale_nodes.rst

```diff
@@ -0,0 +1,360 @@
+ORO Application Scaling - Set-up Instructions
+=============================================
+
+Introduction:
+-------------
+
+Slow-down of the server response when application is under heavy loads may result in deterioration of the 
+user-experience and decrease of the user-satisfaction level. Therefore, we have to somehow optimize and speed-up the 
+response time.
+
+The simplest solution is to scale-up web nodes of the application. For example, scaling of just two web nodes can 
+improve the response time twice (at certain amount of request). The article describes how to scale the web-nodes for the 
+application.
+
+Preliminary Recommendations:
+----------------------------
+
+- Share the *[Application root]/app/cache* folder among all the web nodes, for example using NFS.
+
+- Use Nginx (http://nginx.org/) as a proxy balancer
+
+- Use REDIS (http://redis.io/) as a data cache for Annotations, Metadata, ACL, etc.
+
+- Share *[Application root]/app/attachment* folder among all the web nodes 
+
+.. hint::
+
+    The ways to share *[Application root]/app/attachment* are described in the Attachments section hereinafter. 
+
+Additional Requirements to the Bundles. Known Issues and Solutions:
+-------------------------------------------------------------------
+
+- **Maintenance mode**: there shouldn't be any problems with operations that require the maintenance mode, as default 
+  configuration settings place the 'maintenance_lock' file in the cache directory, which is already shared among the 
+  web-nodes.
+  If you want to store the lock file in another location, see the `documentation on the 
+  LexicMaintenanceBundle <https://github.com/lexik/LexikMaintenanceBundle/blob/master/Resources/doc/index.md>`_.
+
+- **CRON** jobs: there shouldn't be any problems, as the queue is stored in the Database, so all the web-nodes will 
```

Comment:
We had some issue with JMS Job Queue and multiple job daemons, could you please confirm that we can run multiple daemons on multiple nodes? Also as cronjob does jobs scheduling, please confirm that we'll not have duplicates